packages/app-mobile/components/biometrics/sensorInfo.ts

```diff
@@ -42,14 +42,25 @@ export default async (): Promise<SensorInfo> => {
 		// `isSensorAvailable()` is pretty much useless. Instead we ask for
 		// fingerprint when the user turns on the feature and at that point we
 		// know if the device supports biometrics or not.
-		const result = await FingerprintScanner.isSensorAvailable();
-		logger.info('isSensorAvailable result', result);
-		supportedSensors = result;
+		const hasSensor = await hasHardwareAsync();
+		const supportedSensors = (await supportedAuthenticationTypesAsync()).map(sensor => {
+			if (sensor === AuthenticationType.FINGERPRINT) {
+				return 'Touch ID';
+			} else if (sensor === AuthenticationType.FACIAL_RECOGNITION) {
+				return 'Face ID';
+			} else if (sensor === AuthenticationType.IRIS) {
+				return 'Iris';
+			} else {
+				return 'Other';
+			}
+		}).join(',');
```

Comment:
This mapping is intended to produce results that match the previous value of `'security.biometricsSupportedSensors'`.